packages/runtime/test/transport/relationshipTemplates.test.ts

```diff
@@ -287,7 +291,23 @@ describe("RelationshipTemplates query", () => {
             .addStringSet("createdBy")
             .addStringSet("createdByDevice")
             .addNumberSet("maxNumberOfAllocations")
-            .addStringSet("forIdentity");
+            .addStringSet("forIdentity")
+            .addSingleCondition({
+                expectedResult: true,
+                key: "passwordProtection",
+                value: ""
+            })
```

Comment:
where's the negative matcher?

Reply:
I wanted to comment that this test fails - how is it supposed to look like?

Reply:
What's the error?

Reply:
me forgetting to build schemas again 🤦